ROADMAP.md

```diff
@@ -31,4 +31,11 @@ and GetBlockHeaderByHashVerbose methods respectively to follow RPCClient
 naming convention. No functional changes implied.
 
 Removal of GetBlockHeader and GetBlockHeaderVerbose methods is scheduled for
-0.112.0 release.
\ No newline at end of file
+0.112.0 release.
+
+## SessionExpirationTime in RPC configuration
+
+int type SessionExpirationTime has been replaced by string type SessionLifetime
+in the RPC configuration.
```

Comment:
RPC server configuration setting SessionExpiratoinTime of `int` type has been replaced by SessionLifetime of `Duration` type.